doc/functions.xml

```diff
@@ -221,16 +221,68 @@
 
   <para>
     All generators follow a similar call interface: <code>generatorName
-    configFunctions data</code>, where <literal>configFunctions</literal> is a
-    set of user-defined functions that format variable parts of the content.
+    { configFunctions } data</code>, where <literal>configFunctions</literal> is
```

Comment:
Really? As I can see this doesn't seem to be the case, it's still like `toINI { mkSectionName = ... }` and not `toINI { configFunctions.mkSectionName = ... }`, did I miss something? And if yes, then it's a backwards-incompatibility and should have a release notes entry.

Reply:
Ah, is that confusing? I added the `{}` to make it easier to see that it’s an attrset, which is of course not the formally correct way. If it tripped you up, I think I’ll revert that.